layers/+distribution/spacemacs-base/config.el

```diff
@@ -203,7 +203,7 @@ It runs `tabulated-list-revert-hook', then calls `tabulated-list-print'."
               (null dotspacemacs-auto-save-file-location))
     (make-directory autosave-dir t)))
 ;; Choose auto-save location
-(case dotspacemacs-auto-save-file-location
+(pcase dotspacemacs-auto-save-file-location
```

Comment:
mmm... yes, it should be `cl-case` instead of `pcase`. Thanks for your review. I'll resend this patch.